tests/network/utils.py

```diff
@@ -307,11 +318,36 @@ def assert_authentication_request(vm, service_app_name):
     request_response = authentication_request(
         vm=vm,
         service=service_app_name,
-        expected_output=expected_output,
     )
     assert_service_mesh_request(expected_output=expected_output, request_response=request_response)
 
 
+def exec_console_command(vm, command, timeout=TIMEOUT_1MIN):
+    """
+    Execute a command in VM console and return the output.
+
+    Args:
+        vm (VirtualMachine): VM to be used for console connection.
+        command (str): Command to execute.
+        timeout (int, default=TIMEOUT_1MIN): Timeout for the command execution.
+
+    Returns:
+        str: Command output (child.before.decode('utf-8')).
+    """
```

Comment:
Keep in mind that this part of the change will probably need to be removed after [@EdDev 's change](https://github.com/RedHatQE/openshift-virtualization-tests/pull/1588/files#diff-6daf2198793af420d302c77791bd16f97231df3b9178bdc41b7dd5ae8c48c8a2R317) is merged.